frontend/src/utils/formatting.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This is thin wrapper on top of the formatter that's passed to
+ * reat-timeago. The desired date is a format that's something like
+ *
+ * "X years and X days"
+ *
+ * which isn't supported by react-timeago out of the box.
+ *
+ * We add a little more custom logic to format calculate what
+ * the number of days is since the input date, and
+ * then append a suffix to pluralize number of days.
+ *
+ * See more on the formatting options, here:
+ * https://github.com/nmn/react-timeago#formatter-optional
+ */
+const yearAndDayFormatter = (
+  value,
+  unit,
+  suffix,
+  elapsedMilliseconds,
+  /* eslint-disable-next-line no-unused-vars */
+  defaultFormatter,
+  now
+) => {
+  // Only apply custom logic for the year case
+  if (unit === "year") {
+    // The days calculation comes directly from react-timeago:
+    // https://github.com/nmn/react-timeago/blob/master/src/formatters/buildFormatter.js#L84-L86
+
+    // Number of ms in a second: 1000
+    // Number of seconds in a minute: 60
+    // Number of minutes in an hour: 60
+    // Number of hours in a day: 24
```

Comment:
Nit: I like comments, but some code is self explanatory. This could be concise comment.

Reply:
Great call out 🙏 agree this is overly verbose. I'll condense - thanks for highlighting!